utils/codepointtrie/builder/cpp/Makefile

```diff
@@ -0,0 +1,34 @@
+# This file is part of ICU4X. For terms of use, please see the file
```

Comment:
This Makefile looks simple in retrospect, but getting it to work correctly is where I spent the majority of my time